include/executor.hpp

```diff
@@ -149,6 +149,18 @@ class Executor
         latestSrcAndHexwords = pelEventId;
     }
 
+    /**
+     * @brief API to execute functions on requests from d-bus
+     * This method is called whenever there is an ExecuteFunction request from
+     * d-bus.
+     *
+     * @param[in] funcNum - Function number.
+     *
+     * @return Execution status(success/failure, display line 1, display line2)
+     */
+    types::FunctionExecutionResult
```

Comment:
Would suggest RetunStatus  instead of FunctionExecutionResult

Reply:
done